feat: whiskers

uncenter opened the pull request:
Round two!

uncenter committed:
refactor: use whiskers

uncenter committed:
docs: update usage and faq

uncenter committed:
fix: update output location and zipping commands

uncenter committed:
chore: delete pre-zip dir after zipping

uncenter committed:
chore: add new zips

uncenter committed:
refactor(justfile): don't use the same variable for two different thigns

uncenter committed:
chore: crx

uncenter committed:
refactor: build zips instead of crx

uncenter committed:
docs: update

uncenter committed:
chore: delete zips

uncenter committed:
docs: update to mention latest release

uncenter committed:
refactor(justfile): use normal bash for loop

sgoudham commented:
What was the blocker on this again? The CRX signing stuff?

uncenter commented:
> What was the blocker on this again? The CRX signing stuff?

That was it yep, but now we decided to skip the signing and the instructions now say to enable developer mode and load the extension as unpacked.

sgoudham commented on templates/manifest.tera:
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Catppuccin for Opera GX",
+  "description": "Soothing pastel theme for Opera GX",
+  "version": "1.0",
```
Can we adjust this version to `1.0.0` just to match our releases elsewhere in the org?

sgoudham reviewed (commented):
It makes sense to have the zips as release artifacts instead of the source code, yeah?

Not sure if you meant to commit the zips

uncenter commented:
Oh whoops I misremembered which one of these PRs I was supposed to commit zips to source for. I think there was one where we did think about doing it that way? Anyway, reverting the addition of those...

uncenter committed:
fix(templates/manifest): use full semver version & bump to v2.0.0

uncenter committed:
refactor!: build themes/zips to dist/

Directory is already in the gitignore and I think it actually makes sense *not* to use the themes/ directory like we normally do as a distinction between files that are committed to source vs not.

sgoudham reviewed (approved):
Thanks for your patience on this!
